spring/src/main/java/org/axonframework/spring/config/AxonConfiguration.java

```diff
@@ -52,6 +52,7 @@
  * are available in the Configuration, but not made available as Spring beans by default.
  */
 @org.springframework.context.annotation.Configuration("org.axonframework.spring.config.AxonConfiguration")
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.